src/Core/src/Handlers/Picker/PickerHandler.Android.cs

```diff
@@ -165,5 +167,14 @@ static void Reload(IPickerHandler handler)
 		{
 			handler.PlatformView.UpdatePicker(handler.VirtualView);
 		}
+
+		void PlatformViewLayoutChange(object? sender, Android.Views.View.LayoutChangeEventArgs e)
+		{
+			if (PlatformView != null && VirtualView != null)
+			{
+				PlatformView.SetHeight(PlatformView.Height);
+				PlatformView.UpdateVerticalTextAlignment(VirtualView);
```

Comment:
Instead of calling the Update directly, can you rather invoke the mapper? This way if someone overrides the mapper, their code still gets called.

Reply:
```suggestion
				Handler.UpdateValue(nameof(IEntry.VerticalTextAlignment));
```